script/convert.jl

```diff
@@ -0,0 +1,47 @@
+using Pkg, Pkg.TOML
+
+root = joinpath(@__DIR__, "..")
+
+meta = TOML.parsefile(joinpath(@__DIR__, "Notebooks.toml"))
+meta = meta[ARGS[1]]
+
+path = meta["path"]
+deps = get(meta, "deps", [])
+
+for d in ["Project.toml", "Manifest.toml", ".gitignore"]
+  isfile(joinpath(root, path, d)) && push!(deps, d)
+end
+
+mkpath(joinpath(root, "notebooks", path))
+for dep in deps
+  cp(joinpath(root, path, dep), joinpath(root, "notebooks", path, dep), force = true)
+end
+
+pushfirst!(LOAD_PATH, @__DIR__)
+Pkg.activate(joinpath(root, "notebooks", path))
+
+using Literate
+
+function init_nb(content)
+	content = replace(content, "# using CuArrays" => "## using CuArrays")
+	content = replace(content, "#using CuArrays" => "## using CuArrays")
+	content = "using Pkg; Pkg.activate(\".\"); Pkg.instantiate();\n" * content
```

Comment:
I think it would be nice to use the pkg mode syntax here (it's obviously not valid Julia but should work in an IJulia cell).
```suggestion
	content = "] activate .; instantiate" * content
```